data/autoyast_sle12/dns.xml

```diff
@@ -0,0 +1,1157 @@
+<?xml version="1.0"?>
+<!DOCTYPE profile>
+<profile xmlns="http://www.suse.com/1.0/yast2ns" xmlns:config="http://www.suse.com/1.0/configns">
+<!--
+  sets up a dns and tests it
+  via script run by autoyast_verify.pm
+  - custom local zones
+  - reverse dns mapping
+-->
+  <add-on>
+    <add_on_products config:type="list">
+      <listentry>
+        <media_url>http://download.suse.de/ibs/SUSE:/SLE-12-SP2:/Update/standard</media_url>
```

Comment:
how quickly this will it be forgotten that you hardcoded SP2 in a profile named sle12?